sdk/communication/communication-sms/src/utils/uuid.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { generateUuid } from "@azure/core-http";
+
+// This is used as a workaround to be able to stub generateUuid
```

Comment:
If this is just used for testing should it be moved into the tests folder?

Reply:
It's used as a workaround to enable testing, but it's actually referenced in the smsClient.ts itself
I lifted this code from another azure sdk

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/7d9c3dd89019d83e941a56ddcb0f824be0ef3a38/sdk/tables/data-tables/src/TableClient.ts#L52

Reply:
also:
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/src/utils/uuid.ts

Reply:
Is there any reason why we stub it and don't just use it or hard-code a GUID?

Reply:
I have tried
import * as core-http from "@azure/core-http"
followed by stubbing
sinon.stub(core-http, "generateUuid") but this fails in the browser tests

Reply:
Read the whole thing now and understood :-)